src/ngChosen.ts

```diff
@@ -74,16 +75,25 @@ namespace ngChosen {
                     } else if (this.isEmpty(newValue)) {
                         this.updateState(elem, false, true, true);
                     } else {
-                        this.updateState(elem, false, false, false);
+                        this.updateState(elem, false, (!angular.isUndefined(scope.ngDisabled) && scope.ngDisable), false);
                     }
                 });
             } else {
-                this.updateState(elem, false, false, false);
+                this.updateState(elem, false, (!angular.isUndefined(scope.ngDisabled) && scope.ngDisable), false);
+            }
+
+            if (scope.ngDisabled !== undefined) {
+                scope.$watch("ngDisabled",(newValue, oldValue) => {
+                    if (!angular.isUndefined(newValue) && newValue !== oldValue) {
+                        this.updateState(elem, false, newValue, false);
+                        this.triggerUpdate(elem);
```

Comment:
Is this step necessary, as 'updateState' internally calls the 'triggerUpdate' function.